synapse/lib/webapp.py

```diff
@@ -102,14 +102,14 @@ def getFooByBar(self, bar):
         wapp.main()
 
     '''
-    def __init__(self, **settings):
+    def __init__(self, app_config={}, srv_config={}):
+
         EventBus.__init__(self)
         s_daemon.DmonConf.__init__(self)
-        tornado.web.Application.__init__(self, **settings)
+        tornado.web.Application.__init__(self, **app_config)
 
         self.loop = tornado.ioloop.IOLoop()
-        self.serv = tornado.httpserver.HTTPServer(self)
-
+        self.serv = tornado.httpserver.HTTPServer(self, **srv_config)
```

Comment:
The following are the available HTTP server config settings: http://www.tornadoweb.org/en/stable/httpserver.html#tornado.httpserver.HTTPServer